.github/workflows/release_builds.yml

```diff
@@ -22,9 +22,18 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Push kubernetes-tools image
+      - name: Push kubernetes-tools image to Dockerhub
         run: make push-image BUILD_TAG=${{ steps.extract_tag.outputs.tag }}
       - name: Tag latest to point to most recent release
         run: make tag-release-image-with-latest BUILD_TAG=${{ steps.extract_tag.outputs.tag }}
-      - name: Push kubernetes-tools image build cache
+      - name: Push kubernetes-tools image build cache to Dockerhub
         run: make push-image-cache BUILD_TAG=${{ steps.extract_tag.outputs.tag }}
+      - name: Login to ECR
+        run: make login-ecr
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - name: Build and push image to ECR
+        run: make push-image-ecr BUILD_TAG=${{ steps.extract_tag.outputs.tag }}
+      - name: Push kubernetes-tools image build cache to ECR
+        run: make push-image-cache-ecr BUILD_TAG=${{ steps.extract_tag.outputs.tag }}
```

Comment:
Why are we not tagging the published image with `latest` tag in ECR as it is done for Docker Hub in line 28 above?

Reply:
Fixed, missed it.